production/packaging/azure/auction_service/BUILD

```diff
@@ -25,12 +25,10 @@ load(
 )
 load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
 load("@rules_pkg//pkg:zip.bzl", "pkg_zip")
-load("//:config.bzl", "LOG_ENV_VARS")
 
 pkg_files(
     name = "server_executables",
     srcs = [
-        "bin/init_server_basic",
```

Comment:
This change to remove init_server_basic was made because I realized that the upstream 3.10.0 doesn't have it for auction_service, but later I reverted this change ([63ec1b7](https://github.com/KenGordon/bidding-auction-servers/pull/26/commits/63ec1b721503cd776fe67766211c00b185335a52)). Removal of init_server_basic for was made between 3.1 and 3.4 and we didn't do the removal when we merged 3.4.

I don't know the reason we didn't removed it because we didn't discussed about it, but I like to have init_server_basic because it's consistent among services and a bit easier to do experiments.